lib/dlss/capistrano.rb

```diff
@@ -3,3 +3,5 @@
 require 'capistrano/shared_configs'
 
 load File.expand_path('../capistrano/tasks/ssh.rake', __FILE__)
+load File.expand_path('../capistrano/tasks/sidekiq.rake', __FILE__) unless Rake::Task.task_defined?('sidekiq:restart') ||
+                                                                           Rake::Task.task_defined?('sidekiq:start')
```

Comment:
@cbeer Do you think this is sufficient? I've tested that this works as expected.

Reply:
I don't believe it will be sufficient, since content_search, sul_requests, and exhibits do not appear to define a task named either of these.